contrib/babelfishpg_tsql/src/err_handler.h

```diff
@@ -139,6 +139,7 @@ uint8_t		override_txn_behaviour(PLtsql_stmt *stmt);
 #define SQL_ERROR_10727 10727
 #define SQL_ERROR_10733 10733
 #define SQL_ERROR_10793 10793
+#define SQL_ERROR_11506 11506
```

Comment:
Introducing a new error code also requires more dive deep to get what behaviour it aligns to.
On the top of my head we have:
1. Statement aborting
2. batch aborting
3. transaction aborting
Based on any of these we need to add them appropriately in the switch cases of err_handler.c